json/src/test/java/org/jdbi/v3/json/JsonPluginTest.java

```diff
@@ -13,44 +13,52 @@
  */
 package org.jdbi.v3.json;
 
+import java.lang.reflect.Type;
+
+import org.jdbi.v3.core.Jdbi;
 import org.jdbi.v3.core.config.ConfigRegistry;
 import org.jdbi.v3.core.qualifier.QualifiedType;
 import org.jdbi.v3.testing.junit5.JdbiExtension;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.extension.RegisterExtension;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.ArgumentMatchers.any;
```

Comment:
I applaud this. :-)

Reply:
I rather dislike that mocks make your tests depend on exactly what methods are called in what order, rather than the actual behavior of the implementation...